react/week2/todolist/src/App.js

```diff
@@ -0,0 +1,95 @@
+import './App.css';
+import React, { useEffect, useState } from "react";
+
+const todos = [
+    {
+        id: 1,
+        description: "Get out of bed",
+    },
+    {
+        id: 2,
+        description: "Brush teeth",
+    },
+    {
+        id: 3,
+        description: "Eat breakfast",
+    },
+];
+
+const myTodoList = ["Finish my hyf homework", "Walk the dog", "Do laundry", "Prepare for class"]
+
+function TodoItem(props) {
+    const [todoStatus, setTodoStatus] = useState(true);
+    function checkState() {
+        setTodoStatus(!todoStatus)
+    }
+
+    return (
+        <div className="todo-item">
+            <div className="item-description">
+                <input type="checkbox" onClick={checkState}></input>
+                <li style={{ 'textDecoration': todoStatus ? 'none' : 'line-through' }} >{props.description}</li>
+            </div>
+            <div>
+                <button onClick={() => props.deleteMethod(props.id)}> Delete </button>
+            </div>
+        </div>
+    )
+}
+
+function ListContainer() {
```

Comment:
Think carefully about the names you choose. Names are linked to our understanding.
"ListContainer", "List" and "item" are ambiguous names that dont tell us exactly what they contain.